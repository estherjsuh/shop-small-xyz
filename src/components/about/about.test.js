import React from 'react';
import {shallow} from 'enzyme';
import About from './about.component';


describe('<About/>', () => {
    it('renders `.heading`', () =>{
        const wrapper = shallow(<About/>);
        expect(wrapper.find('.heading'));
    });

    it('renders 4 `.statement`', () =>{
        const wrapper = shallow(<About/>);
        expect(wrapper.find('.statement').length).toEqual(4);
    });

    it('try snapshot', ()=>{
        expect(shallow(<About/>)).toMatchSnapshot();
    })

});