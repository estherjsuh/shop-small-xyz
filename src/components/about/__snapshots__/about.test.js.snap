// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About/> try snapshot 1`] = `ShallowWrapper {}`;
